src/main/java/com/stripe/model/EventRequest.java

```diff
@@ -8,18 +8,19 @@
 @Getter
 @Setter
 @EqualsAndHashCode(callSuper = false)
-public class EventRequest extends StripeObject {
+public class EventRequest extends StripeObject implements HasId {
   /**
-   * ID of the API request that caused the event. If null, the event was automatic (e.g., Stripe’s
-   * automatic subscription handling). Request logs are available in the <a
-   * href="https://dashboard.stripe.com/logs">dashboard</a>, but currently not in the API.
+   * ID of the API request that caused the event. If null, the event was automatic (e.g., Stripe's
+   * automatic subscription handling). Request logs are available in the
+   * [dashboard](https://dashboard.stripe.com/logs), but currently not in the API.
    */
+  @Getter(onMethod_ = {@Override})
```

Comment:
Minor but what does this one change?

Reply:
Practically speaking, nothing.

`EventRequest` now implements the `HasId` interface, and the `getId()` signature is defined in the interface, so concrete implementations need to have the `@Override` annotation.